### ISSUE TITLE
[Question] How can we contribute to the Android Library ?

### ISSUE THREAD

**Issue by phildupuis** [notifee/react-native-notifee#272]
There is currently two opened issues for which we would like to contribute : 
[Feature request: FullScreenIntent (Android)](https://github.com/notifee/react-native-notifee/issues/45)
[Foreground Service notification not removing after resolve](https://github.com/notifee/react-native-notifee/issues/203)

Since these bugs are 2 Android bugs, modifications will most probably be needed on the [aar](https://github.com/notifee/react-native-notifee/tree/master/android/libs). 

Is it possible to get access and contribute to the aar ?

We would be glad to help :) 
Thank you

**Comment by phildupuis** [comment]
Hi @mikehardy 👋 , sorry for pinging you directly. I see that you are a major contributor to the project. Do you have any thoughts on this ?

**Comment by mikehardy** [comment]
Hi @phildupuis ! I saw this and mentioned it internally when you first posted actually ;-), I just raised it internally again. Not speaking Notifee-officially, just as a person that maintains repos this seems like an obvious win/win with (likely) just some relatively easy details to work out, so thanks for pinging me. I will let you know if I hear anything, of course and feel free to ping again if for some reason there's no motion, I can take it up a notch and instead of just posting it on Invertase internal Slack I can do something crazy like a synchronous voice call to get attention ;-)

**Comment by phildupuis** [comment]
Nice ! Thanks a lot @mikehardy

**Comment by phildupuis** [comment]
Hi @mikehardy ! Any news on this ? Time for a synchronous voice call ? 😛

**Comment by phildupuis** [comment]
I just realized that [Feature request: FullScreenIntent (Android)](https://github.com/notifee/react-native-notifee/issues/45) might be solved in this [PR](https://github.com/notifee/react-native-notifee/pull/279) and that [Foreground Service notification not removing after resolve](https://github.com/notifee/react-native-notifee/issues/203) was added to v1.3.0. So, an answer to my question may not be as urgent as it was, but I'm still curious about how we could get access to the aar for solving potential future issues.

**Comment by mikehardy** [comment]
Yeah - we're making steady forward progress internally, but I still consider this a pending item and I think it will have to be some sort of synchronous chat internally to bat around ideas...
